### DISCUSSION TITLE
Batch run with .bat

### DISCUSSION THREAD

**Discussion by trungng2006** [iansan5653/open-mcr#87] [Q&A]
Thank you for sharing this wonderful program.

I want to use open-mcr to grade multiple subject in multiple folders.
Could you please add into the readme how to do batch run with this program?

Thank you so much!

**Comment by iansan5653** [comment]
Hello! You can run the CLI by downloading the files and running them with Python. [Here are instructions on downloading and running the Python code](https://github.com/iansan5653/open-mcr#running-from-source--cli), and then you can run `python3 src/main.py --help` to see the list of arguments. You can also see the list of arguments in [this source file](https://github.com/iansan5653/open-mcr/blob/master/src/main.py).

As a basic example, you might have a batch file that looks like:

```bat 
python3 open-mcr/src/main.py /path/to/input_1 /path/to/output_1 --variant 75
python3 open-mcr/src/main.py /path/to/input_2 /path/to/output_2 --variant 75
```

**Reply by trungng2006** [reply]
Thank you so much!!!
